lib/browser/api/screen.ts

```diff
@@ -16,5 +16,23 @@ export default new Proxy({}, {
       return v.bind(_screen);
     }
     return v;
+  },
+  ownKeys: () => {
+    if (_screen === undefined) {
+      _screen = createScreen();
+    }
+    return Reflect.ownKeys(_screen);
+  },
+  has: (target, prop: string) => {
+    if (_screen === undefined) {
+      _screen = createScreen();
+    }
+    return prop in _screen;
```

Comment:
For completeness should this be `Reflect.has`?

Reply:
That’s [identical](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect/has).